x/pool-incentives/keeper/distr_test.go

```diff
@@ -46,7 +52,12 @@ func (suite *KeeperTestSuite) TestAllocateAsset() {
 	keeper := suite.app.PoolIncentivesKeeper
 	mintKeeper := suite.app.MintKeeper
 	params := suite.app.MintKeeper.GetParams(suite.ctx)
-	params.DeveloperRewardsReceiver = sdk.AccAddress([]byte("addr1---------------")).String()
+	params.WeightedDeveloperRewardsReceivers = []minttypes.WeightedAddress{
```

Comment:
Can we some tests with multiple addresses

Reply:
Added